pkg/codegen/docs.go

```diff
@@ -25,20 +25,26 @@ import (
 type DocLanguageHelper interface {
 	GetPropertyName(p *schema.Property) (string, error)
 	GetEnumName(e *schema.Enum, typeName string) (string, error)
-	GetDocLinkForResourceType(pkg *schema.Package, moduleName, typeName string) string
-	GetDocLinkForPulumiType(pkg *schema.Package, typeName string) string
-	GetDocLinkForResourceInputOrOutputType(pkg *schema.Package, moduleName, typeName string, input bool) string
-	GetDocLinkForFunctionInputOrOutputType(pkg *schema.Package, moduleName, typeName string, input bool) string
-	GetLanguageTypeString(pkg *schema.Package, moduleName string, t schema.Type, input bool) string
-
+	// GetTypeName gets the name of a type in the language of the DocLanguageHelper.
+	//
+	// relativeToModule should always be a module returned from pkg.TokenToModule. It
+	// should not be language specialized.
```

Comment:
Can we give an example here? E.g. "For instance, a call for Python might look like ..." -- it's not clear to me what I should expect `relativeToModule` to look like. Additionally, what does `relativeToModule` do?

Reply:
It is used to control how the returned `name` is qualified. I expended the comment to explain.

For example `"pkg:module:Type"` might be `"Type"` relative to `"module"`, but `"module.Type"` relative to `"index"`.